### DISCUSSION TITLE
Load and write workbook with custom properties

### DISCUSSION THREAD

**Discussion by Nxito** [exceljs/exceljs#2046] [General]
![image](https://user-images.githubusercontent.com/55735074/165717587-1362d880-4824-49df-879d-5ec2006c24fd.png)
How can i load and write this properties? 

Its available on this library?

**Comment by maechler** [comment]
@Nxito I had a quick look at the API and source code and haven't found a way so far. Did you find a way to do it with this library?

**Reply by Nxito** [reply]
Unfortunately i think it was not possible, but we were able to do something alike using the sharepoint platform api

**Reply by maechler** [reply]
Alright, thanks for your feedback!
